src/main/groovy/org/labkey/gradle/plugin/Standalone.groovy

```diff
@@ -0,0 +1,62 @@
+package org.labkey.gradle.plugin
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.labkey.gradle.util.BuildUtils
+
+/**
+ * Use to build modules without platform repository and to build distributions without the server repository
+ * Module will build normally unless the 'standaloneBuild' property is set, in which case, additional plugins will be
+ * applied to allow the module to build distributions outside of a server repository.
+ */
+class Standalone implements Plugin<Project> {
+
+    void apply(Project project) {
+        project.apply plugin: 'org.labkey.build.module'
+
+        if (BuildUtils.isStandaloneBuild(project)) {
+            project.apply plugin: 'org.labkey.build.tomcat'
+            project.apply plugin: 'org.labkey.build.serverDeploy'
+
+            project.allprojects {
+                repositories {
+                    mavenLocal()
+                    maven {
+                        url "${project.artifactory_contextUrl}/ext-tools-local"
+                    }
+                    maven {
+                        url "${project.artifactory_contextUrl}/libs-release"
+                    }
+                    maven {
+                        url "${project.artifactory_contextUrl}/libs-snapshot"
+                    }
+                    jcenter()
+                }
+            }
```

Comment:
I think it's not a good idea to have these repositories declared in the plugin.  Some users will want/need to declare credentials, for example.  In any case, jcenter should be removed since it's going away and there should be a check for the existence of `artifactory_contextUrl`.

Reply:
Ok. I had a feeling that might be the case. I removed the repository declaration and added it to the module.